infra/bridge/src/controllers/identity.js

```diff
@@ -82,7 +84,8 @@ router.get('/', identityReadVerify, async (req, res) => {
  * @args {string} req.body.ipfsHash: IPFS hash of the identity JSON blob.
  * @returns {{id: string}} Returns the owner address used to write the identity to the DB.
  */
-router.post('/', identityWriteVerify, async (req, res) => {
+router.post('/', authMiddleware, identityWriteVerify, async (req, res) => {
+  const authAddress = req.__originAuth.address.toLowerCase()
   const ethAddress = req.query.ethAddress
```

Comment:
nit: Can we just use `authAddress` and not require `ethAddress` to be sent in the query params?

Reply:
Yes that's a very good idea. I have added a TODO for now and will take care of this in a separate PR once we are sure the auth integration is working.